join.html

```diff
@@ -181,7 +181,7 @@ <h2 class="progress">Step 1 of 4: Personal Details</h2>
             type="text"
             name="city"
             id="city"
-            placeholder="Enter your city"
+            placeholder="e.g Bangaluru"
```

Comment:
Typo in the name here.

Reply:
@ankushdharkar thanks for the review will make the change 😄